pkg/store/proxy.go

```diff
@@ -81,38 +81,73 @@ func (s *ProxyStore) Info(ctx context.Context, r *storepb.InfoRequest) (*storepb
 		StoreType: s.component.ToProto(),
 	}
 
-	MinTime := int64(math.MaxInt64)
-	MaxTime := int64(0)
-
+	minTime := int64(math.MaxInt64)
+	maxTime := int64(0)
 	stores := s.stores()
+
+	if len(stores) == 0 {
+		res.MaxTime = math.MaxInt64
+		res.MinTime = 0
+
+		return res, nil
+	}
+
 	for _, s := range stores {
 		mint, maxt := s.TimeRange()
-		if mint < MinTime {
-			MinTime = mint
+		if mint < minTime {
+			minTime = mint
 		}
-		if maxt > MaxTime {
-			MaxTime = maxt
+		if maxt > maxTime {
+			maxTime = maxt
 		}
 	}
 
-	// Edge case: we have all of the data if there are no stores.
-	if len(stores) == 0 {
-		MinTime = 0
-		MaxTime = math.MaxInt64
-	}
-
-	res.MaxTime = MaxTime
-	res.MinTime = MinTime
+	res.MaxTime = maxTime
+	res.MinTime = minTime
 
 	for _, l := range s.selectorLabels {
```

Comment:
I assume we are not removing `selector Labels` for now?

Reply:
It somewhat depends on our decision around the changelog. If we decide all queriers must be >=v0.6.0 if any are, then this would be safe to remove.

Reply:
:+1:  Let's leave for now